docs/admin/executors/deploy_executors_kubernetes.mdx

```diff
@@ -1,67 +1,48 @@
-# Deploying Sourcegraph executors natively on Kubernetes
+# Deploying Sourcegraph Executors natively on Kubernetes
 
 <Callout type="note">This feature is in beta and is available in Sourcegraph 5.1.0 and later.</Callout>
 
-[Kubernetes manifests](https://github.com/sourcegraph/deploy-sourcegraph-k8s) are provided to deploy Sourcegraph Executors on a running Kubernetes cluster. If you are deploying Sourcegraph with helm, charts are available [here](https://github.com/sourcegraph/deploy-sourcegraph-helm).
+The native Kubernetes Executors have a master Executor pod that schedules worker pods via the Kubernetes API. The master Executor pod manages the lifecycle of jobs, while the worker pods process the actual [batch change](/batch_changes/explanations/server_side) or [precise auto indexing](/code-search/code-navigation/auto_indexing) job specs. For more details, see [how it works](/admin/executors#native-kubernetes).
 
 ## Requirements
 
-### Feature flag
-To instruct Sourcegraph to use Kubernetes-deployed executors, you will need to enable the `native-ssbc-execution` [feature flag](/admin/executors/native_execution#enable).
-
-### RBAC Roles
-
-Executors interact with the Kubernetes API to handle jobs. The following are the RBAC Roles needed to run Executors on
-Kubernetes.
+Executors interact with the Kubernetes API to manage the lifecycle of individual executor jobs. The following are the RBAC Roles needed to run Executors natively on Kubernetes.
 
 | API Groups |     Resources      |         Verbs          |                                          Reason                                           |
 | ---------- | ------------------ | ---------------------- | ----------------------------------------------------------------------------------------- |
 | `batch`    | `jobs`             | `create`, `delete`     | Executors create Job pods to run processes. Once Jobs are completed, they are cleaned up. |
 |            | `pods`, `pods/log` | `get`, `list`, `watch` | Executors need to look up and steam logs from the Job Pods.                               |
 
-See the [example Role YAML](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@main/-/blob/cmd/executor/kubernetes/batches/executor-batches.Role.yml)
-for details.
+See the [example Role YAML](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@main/-/blob/cmd/executor/kubernetes/batches/executor-batches.Role.yml) for more details on the exact RBAC requirements for native Kubernetes Executors.
```

Comment:
Examples are awesome. Can we sprinkle more of them throughout the docs? Maybe links to deployment repos?

Reply:
There are links to the deployment repos just below this on lines 20 & 33

Reply:
Btw these examples were created when we first introduced k8s executors

Reply:
> created when we first introduced k8s executors
Are they out of date?

Reply:
> Are they out of date?

I was wondering the same thing, but there's been very little change to the k8s executor deployment method so I think they are fine. I see more risk with them drifting over time in the future, since there's no clear owner to keep these maintained.